packages/workbox-google-analytics/index.mjs

```diff
@@ -0,0 +1,172 @@
+/*
+ Copyright 2017 Google Inc. All Rights Reserved.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+
+/**
+ * @module workbox-google-analytics
+ */
+
+import {Queue, QueuePlugin} from 'workbox-background-sync';
```

Comment:
It's a shame that to use this module a developer will need to import, core, backgroundsync, routing and runtime-caching to work.

Reply:
I think it's only a shame if it means someone who only wants this module gets all the extra stuff in routing and runtime caching. That's why I was advocating for the ability to do imports like this:

```js
import Route from 'workbox-routing/Route.mjs';
import Router from 'workbox-routing/Router.mjs';
```

instead of this:

```js
import {Route, Router} from 'workbox-routing';
``

Assuming we can get it to work only importing what it needs then I don't consider it a shame, I consider it a win.

I mean, I could easily implement this myself in this package, but isn't the point that users who want multiple workbox packages now get a minimal footprint? And they get things like consistent logging out of the box?